### PR TITLE
Add ability to edit CfP

### LINKED ISSUE #185: Add ability to edit CfP
Any one can submit CfP: `http://127.0.0.1:8000/<slug>/cfp/` it creates speaker and talk. Create new view that will allow edit submitted data.

New view should be available via url: `http://127.0.0.1:8000/<slug>/cfp/<uuid>`. We should search talk uuid and return form so speaker can edit already submitted data. Edit should be available only if status is CFP or DRAFT otherwise show 404 page.

### PR DESCRIPTION
**Pull request by ZoltanOnody**
close #185

### CODE REVIEW

File: konfera/event/views.py [hunk 1]
```diff
@@ -53,30 +55,65 @@ def event_details_view(request, slug):
...
-        speaker_instance = speaker_form.save()
-        talk_instance = talk_form.save(commit=False)
-        talk_instance.primary_speaker = speaker_instance
-        talk_instance.status = Talk.CFP
-        talk_instance.event = Event.objects.get(slug=slug)
-        talk_instance.save()
-        message_text = _("Your talk proposal successfully created.")
-        messages.success(request, message_text)
+    def dispatch(self, *args, **kwargs):
+        get_object_or_404(Event, slug=kwargs.get('slug'))
```
Comment (ricco386) [inline konfera/event/views.py]: please add blank line for improved readability

Comment (ZoltanOnody) [inline konfera/event/views.py]: above or bellow this (`get_object_or_404(Event, slug=kwargs.get('slug'))`) line?

Comment (ricco386) [inline konfera/event/views.py]: below, so there is blank line before return

**Comment by AliaTabassumAS** [comment]
.

**Comment by ricco386** [comment]
@ZoltanOnody thank you

**Pull request merged**